backend/forecastbox/api/routers/product.py

```diff
@@ -59,38 +63,56 @@ def select_from_params(available_spec: Qube, params: dict[str, Any]) -> Qube:
 
     return available_spec
 
+def _sort_values(values: list[Any]) -> list[Any]:
+    if all(str(x).isdigit() for x in values):
+        values = list(map(float, sorted(values, key=float)))
+        return values
+    values = list(sorted(values, key=lambda x: str(x).lower()))
+    return values
+
+def _sort_fields(fields: dict) -> dict:
+    new_fields = OrderedDict()
+    for key in CONFIG_ORDER:
+        if key in fields:
+            new_fields[key] = fields[key]
+
+    for key in fields:
+        if key not in new_fields:
+            new_fields[key] = fields[key]
+
+    return new_fields
```

Comment:
just be aware that dictionaries dont preserve order when serialized... list of tuples safer